Alignment/RPData/LHC/2015_10_18_fill4511/version2/sr+el/45.xml

```diff
@@ -0,0 +1,115 @@
+<!--
```

Comment:
@ianna @fwyzard @jan-kaspar I've remove this problematic files - there were just examples of alignment data. As we not have them yet in the database, we need some examples to run and test the code.